cfgov/v1/atomic_elements/organisms.py

```diff
@@ -781,73 +780,41 @@ class Expandable(BaseExpandable):
-    body = blocks.RichTextBlock(required=False)
-    is_accordion = blocks.BooleanBlock(required=False)
-    has_top_rule_line = blocks.BooleanBlock(
-        default=False,
-        required=False,
-        help_text=('Check this to add a horizontal rule line to top of '
-                   'expandable group.')
-    )
-
-    expandables = blocks.ListBlock(Expandable())
+class ContactExpandable(blocks.StructBlock):
+    contact = SnippetChooserBlock('v1.Contact')
 
     class Meta:
-        icon = 'list-ul'
-        template = '_includes/organisms/expandable-group.html'
+        icon = 'user'
+        template = '_includes/organisms/contact-expandable.html'
 
     class Media:
-        js = ["expandable-group.js"]
+        js = ['expandable.js']
 
 
 class ContactExpandableGroup(blocks.StructBlock):
-    """Expandable group that renders selected Contact snippets."""
-    group_title = blocks.CharBlock()
-    contacts = blocks.ListBlock(SnippetChooserBlock('v1.Contact'))
-
-    def get_context(self, value, parent_context=None):
-        context = super(ContactExpandableGroup, self).get_context(
-            value,
-            parent_context=parent_context
-        )
-
-        # This block mimics the ExpandableGroup block.
-        context['value'] = ExpandableGroup().to_python({
-            'heading': value['group_title'],
-            'expandables': [
-                {
-                    'label': contact.heading,
-                    'content': contact.contact_info.stream_data,
-                } for contact in value['contacts']
-            ],
-        })
-
-        return context
+    heading = blocks.CharBlock(required=False)
+    expandables = blocks.ListBlock(ContactExpandable())
 
-    def bulk_to_python(self, values):
-        """Support bulk retrieval of Contacts to reduce database queries.
+    class Meta:
+        icon = 'list-ul'
+        template = '_includes/organisms/expandable-group.html'
 
-        This method leverages Wagtail's undocumented method for doing bulk
-        retrieval of data for StreamField blocks. It retrieves all Contacts
-        referenced by a StreamField in a single lookup, instead of the
-        default behavior of doing one database query per SnippetChooserBlock.
-        """
-        contact_ids = set(
-            itertools.chain(*(block['contacts'] for block in values))
-        )
+    class Media:
+        js = ['expandable-group.js']
 
-        contact_model = self.child_blocks['contacts'].child_block.target_model
-        contacts_by_id = contact_model.objects.in_bulk(contact_ids)
 
-        for block in values:
-            block['contacts'] = [
-                contacts_by_id[contact_id]
-                for contact_id in block['contacts']
-            ]
+class ExpandableGroup(blocks.StructBlock):
+    heading = blocks.CharBlock(required=False)
+    body = blocks.RichTextBlock(required=False)
+    is_accordion = blocks.BooleanBlock(required=False)
```

Comment:
Maybe we should exclude accordion while its status is up in the air, or is it better to have here for completeness?

Reply:
Same comment as above; I'm not sure why this is showing up as a change.